client/blindai/utils/errors.py

```diff
@@ -8,6 +8,9 @@ def check_rpc_exception(rpc_error):
     elif rpc_error.code() == grpc.StatusCode.UNAVAILABLE:
         return f"Failed to connect to GRPC server: code={rpc_error.code()} message={rpc_error.details()}"
 
+    elif rpc_error.code() == grpc.StatusCode.UNIMPLEMENTED:
```

Comment:
Why are you using this error code?

Reply:
If we try to connect with the server 0.1.0 for example with this client, the ``stub.GetServerInfo`` call will raise a rgc_error with unimplemented code, as GetServerInfo is not implemented in that version on the server, and this exception will thus be raised whenever the client is calling an RPC procedure that is not implemented by the server, which means the versions are not compatible.

Reply:
Cool. Clear.